src/actions/uploadAndCloudinary.ts

```diff
@@ -0,0 +1,28 @@
+import { FleekUploadedFile, UploadedFile } from "../storage";
+import { getCloudinaryStorage, getFleekStorage } from "./helpers";
+
+export async function uploadContentToIPFSAndCloudinary(
```

Comment:
function names that contain "and" usually point to a problem even if they're two things are similar in function

I would argue that a single "uploadContent" function that calls out independently to an "uploadToIPFS" and also to an "uploadToCloudinary" would be an improvement

Reply:
Also this filename is a bit confusing to me, "uploadAndCloudinary.ts" should be changed

Reply:
I was a bit concerned about having one action call another action, since they are supposed to both be independent workstreams, but I think it should be okay to do that for now.